.gitignore

```diff
@@ -11,3 +11,5 @@ system-test/*key.json
 .DS_Store
 google-cloud-logging-winston-*.tgz
 google-cloud-logging-bunyan-*.tgz
+package-lock.json
```

Comment:
You don't need the `package-lock.json` with the `**/package-lock.json` below :)

Reply:
Done